syntax/supercollider.vim

```diff
@@ -39,6 +39,7 @@ syn match	scSymbol "\w\+:"
 
 syn region  scString start=+"+ skip=+\\\\\|\\"+ end=+"+
 
+syn match	scChar	"\$[^ ]"
```

Comment:
So i don't think deleting the lines below will work, because `$<space>` and `$\<space>` (and possibly also `$\<non-ws>`) won't be caught anymore.